lib/ansible/config/module_defaults.yml

```diff
@@ -270,6 +282,8 @@ groupings:
   - aws
   elb_target:
   - aws
+  elb_target_facts:
```

Comment:
elb_target_info now (modules that do not return ansible_facts should not use the _facts suffix)

Reply:
updated